### PR TITLE
Creating specs for admin model

### PR DESCRIPTION
**Pull request by sebjimenez**
# Purpose or Intent

- The goal for this PR is to create **specs** for the **admin model.**

### CODE REVIEW

**Comment by sebjimenez** [comment]
Ready to **review** @sergio-ocon. Waiting for feedback

**Comment by sergio-ocon** [comment]
Please review code for User to see how to do this properly

**Comment by sergio-ocon** [comment]
https://github.com/propedeutica/shift-reservation/pull/53

**Comment by sebjimenez** [comment]
This PR is ready to **review**. Please **download branch and provide feedback.** @sergio-ocon @amaurygonzalez @agarciavera

**Comment by agarciavera** [comment]
I like these rspec except the little part to review. LGTM.

File: spec/models/admin_spec.rb [hunk 1]
```diff
@@ -1,5 +1,76 @@
...
+  it "is invalid with a too short email" do
+    admin.email = "a"
+    admin.valid?
+    expect(admin.errors[:email]).to include(I18n.t('email.invalid', count: 0, scope: active_record_admin))
+  end
+
+  it "is invalid with a too long email" do
+    admin.email = "a" * 255 + "b"
+    admin.valid?
+    expect(admin.errors[:email]).to include(I18n.t('email.invalid', count: 255, scope: active_record_admin))
```
Comment (sergio-ocon) [inline spec/models/admin_spec.rb]: Can you use .invalid?

Comment (sebjimenez) [inline spec/models/admin_spec.rb]: You already merged, 5 minutes ago, have you corrected it? Sorry I didn't saw it as soon as you commented.

**Pull request merged**